src/push.rs

```diff
@@ -80,6 +80,10 @@ fn push(job: &str,
         method: &str)
         -> Result<()> {
 
+    // Suppress clippy warning needless_pass_by_value.
+    let grouping = grouping;
+    let mfs = mfs;
```

Comment:
Can we use `AsRef` to accept both value and reference? https://doc.rust-lang.org/std/convert/trait.AsRef.html